packages/gatsby-theme-carbon/src/components/NextPrevious/NextPrevious.module.scss

```diff
@@ -0,0 +1,45 @@
+@import '~carbon-components/scss/globals/scss/vars';
+@import '~@carbon/elements/scss/type/styles';
+
+.wrapper {
+  background: $carbon--gray-90;
+  color: $carbon--white-0;
+}
+
+.link-container {
+  min-height: rem(128px);
+  display: flex;
+}
+
+.link-container .link {
+  color: $carbon--white-0;
+  padding: 1rem;
+  text-decoration: none;
+  width: 50%;
+  transition: background $duration--moderate-02 $carbon--standard-easing;
+  &:hover {
+    background: $carbon--gray-80;
+  }
+}
+
+.first-link {
+  margin-right: auto;
+}
+
+.second-link {
+  margin-left: auto;
+}
+
+.direction {
+  @include carbon--type-style('body-short-01');
+}
+
+.name {
+  @include carbon--type-style('expressive-heading-03', true);
+  padding-bottom: 3rem;
```

Comment:
Do we need this padding-bottom? It's making the buttons too tall at mobile breakpoints. (should also be `$spacing-09` for 3rem)